.travis.yml

```diff
@@ -85,4 +85,5 @@ script:
   - codecov
 
 after_script:
+  - npm run vscode:package
```

Comment:
I don't think this can be in the after_script.

See https://docs.travis-ci.com/user/customizing-the-build/#Breaking-the-Build 

    The exit code of after_success, after_failure, after_script and subsequent stages do not affect the build result. However, if one of these stages times out, the build is marked as a failure.

We need this to fail the build so it cannot be in the `after_script`.

I'm less with familiar with appveyor but I suspect that you shouldn't put it under the `on_finish` as well.

Was there a reason you chose to put it there?

Reply:
Didn't know failures in after_script don't affect build results. Just thought it'd be a good place for a final sanity check. I'll move both to the script areas.